### PR TITLE
README: print -> println

### PR DESCRIPTION
**Pull request by n1kio**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -812,7 +812,7 @@ Use `not=` instead of `(not (= ...))`.
 
 === `printf` [[printf]]
 
-Use `printf` instead of `(print (format ...))`.
+Use `printf` instead of `(println (format ...))`.
```
Comment (bbatsov) [inline README.adoc]: Does `printf` insert a newline?

Comment (n1kio) [reply README.adoc]: No, I chose it for consistency with the example following.

Comment (bbatsov) [reply README.adoc]: Ah, I see what you mean now. The rule was meant to imply a generic combination of `print(ln` and `format`, so I'd go with some different wording instead.

Comment (n1kio) [reply README.adoc]: I see, in that case it might be better to show both.

**Pull request closed without merging**